src/main/java/org/generationcp/breeding/manager/listmanager/listeners/AddColumnMenuItemClickListener.java

```diff
@@ -185,9 +185,9 @@ void addCrossMalePrefNameColumn() {
 	}
 
 	void addCrossFemaleGidColumn() {
-		if (!this.addColumnSource.columnExists(ColumnLabels.CROSS_FEMALE_GID.getName())) {
-			this.addColumnSource.addColumn(ColumnLabels.CROSS_FEMALE_GID);
-			this.valuesGenerator.setCrossFemaleInfoColumnValues(ColumnLabels.CROSS_FEMALE_GID.getName(),
+		if (!this.addColumnSource.columnExists(ColumnLabels.FGID.getName())) {
+			this.addColumnSource.addColumn(ColumnLabels.FGID);
+			this.valuesGenerator.setCrossFemaleInfoColumnValues(ColumnLabels.FGID.getName(),
 					FillWithOption.FILL_WITH_CROSS_FEMALE_GID);
```

Comment:
I think you also need to change the ColumnLabels value for the following items in `FillWithOption` enum:
```
FILL_WITH_CROSS_FEMALE_GID
FILL_WITH_CROSS_MALE_GID
```

Reply:
Hi @darla-leafnode , 
Noted. Code Updated. 
Thanks
Gelli